charter.md

```diff
@@ -16,15 +16,11 @@ select and approve access to devices as opposed to assuming certification and in
 
 ## Scope of Work
 
-{TBD: Describe topics that are in scope.
-For specifications that the CLA patent section applies to,
-it is helpful to describe the scope in a way that it is clear
-what types of technologies will be defined in specifications,
-as opposed to adoption by reference or underlying technology not defined in the proposed spec.
-Key use cases are often helpful in describing scope.
-If no specifications will be defined in the group that the CLA patent section applies to,
-the charter should clearly state that.
-A clear scope is particularly important where patent licensing obligations may apply.}
+The [Web Bluetooth Use Cases](http://webbluetoothcg.github.io/web-bluetooth/use-cases.html)
```

Comment:
Do we need to copy the relevant use cases into the charter, since the use cases document isn't as frozen as the charter?